src/components/person/Person.jsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+
+export const Person = ({ person }) => {
+  const { name, age, sex, isMarried, partnerName } = person;
+
+  return (
+    <section className="Person">
+      <h2 className="Person__name">{`My name is ${name}`}</h2>
+      <p className="Person__age">{age && (`I am ${age}`)}</p>
```

Comment:
у твоєму варіанті якщо немає віку ти рендериш пустий тег `p`, краще взагалі його не рендерити
```suggestion
       {age && (
          <p className="Person__age">{`I am ${age}`)</p>
       )}
```